api/user.proto

```diff
@@ -128,6 +128,11 @@ message UpdateUserPasswordRequest {
   bool revoke_all_access = 4;
 }
 
+message TransferUserRightsRequest {
+  UserIdentifiers user_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
+  UserIdentifiers receiver_ids = 2 [(gogoproto.nullable) = false, (validate.rules).message.required = true];
```

Comment:
```suggestion
  UserIdentifiers sender_ids = 1 [(gogoproto.nullable) = false, (validate.rules).message.required = true];
  UserIdentifiers receiver_ids = 2 [(gogoproto.nullable) = false, (validate.rules).message.required = true];
```

I'm actually not a big fan of `sender`/`receiver` (you don't send rights, you transfer them), and would prefer `from`/`to` (you transfer rights from user A to user B)